docs/api/target_properties.rst

```diff
@@ -155,12 +155,24 @@ blt_print_target_properties
 
 .. code-block:: cmake
 
-    blt_print_target_properties(TARGET <target>)
+    blt_print_target_properties(TARGET               <target name>
+                                CHILDREN             <TRUE|FALSE>
+                                PROPERTY_NAME_REGEX  <regular_expression_string>
+                                PROPERTY_VALUE_REGEX <regular_expression_string>)
 
 Prints out all properties of the given target.
```

Comment:
```suggestion
Prints out all properties of the given target. Can also print target's dependencies with the 
`CHILDREN` argument, as well as specific filter properties using regular expressions with 
`PROPERTY_NAME_REGEX` and `PROPERTY_VALUE_REGEX`.
```

Reply:
I feel like I already explain what the arguments are [below that line](https://github.com/LLNL/blt/pull/590/files/401f967a4e0e87b83ba09b914cbd88fdddbeefc9#diff-b74b6950ded9a2c4317c89df885955d3a1907c5dafbdee8fc288c1029cf24a5cR168-R175)

Reply:
well "of a given target" is not entirely accurate anymore for a brief description.

Reply:
It could also just be something like:

Prints out all, or filtered, properties of a given target with optionally printing dependencies.